gateway/src/apicast/executor.lua

```diff
@@ -38,7 +38,10 @@ local function build_context(executor)
 end
 
 local function shared_build_context(executor)
-    local ctx = ngx.ctx or {}
+    local ctx, err = pcall(function() return ngx.ctx end) or {}
```

Comment:
This does not work. `pcall` returns `true/false` as a first value.

>  Its first result is the status code (a boolean), which is true if the call succeeds without errors
http://www.lua.org/manual/5.2/manual.html#pdf-pcall